src/utils.ts

```diff
@@ -1,81 +1,100 @@
+import * as fs from 'fs';
+import * as core from '@actions/core';
 import * as dockleHelper from './dockleHelper';
 import * as gitHubHelper from './gitHubHelper';
 import * as inputHelper from './inputHelper';
 import * as trivyHelper from './trivyHelper';
 
 export function getCheckRunPayloadWithScanResult(trivyStatus: number, dockleStatus: number): any {
-    const headSha = gitHubHelper.getHeadSha();
-    const checkConclusion = getCheckConclusion(trivyStatus, dockleStatus);
-    const checkSummary = getCheckSummary(trivyStatus, dockleStatus);
-    const checkText = getCheckText(trivyStatus, dockleStatus);
+  const headSha = gitHubHelper.getHeadSha();
+  const checkConclusion = getCheckConclusion(trivyStatus, dockleStatus);
+  const checkSummary = getCheckSummary(trivyStatus, dockleStatus);
+  const checkText = getCheckText(trivyStatus, dockleStatus);
 
-    const checkRunPayload = {
-        head_sha: headSha,
-        name: `[container-scan] ${inputHelper.imageName}`,
-        status: "completed",
-        conclusion: checkConclusion,
-        output: {
-            title: "Container scan result",
-            summary: checkSummary,
-            text: checkText
-        }
+  const checkRunPayload = {
+    head_sha: headSha,
+    name: `[container-scan] ${inputHelper.imageName}`,
+    status: "completed",
+    conclusion: checkConclusion,
+    output: {
+      title: "Container scan result",
+      summary: checkSummary,
+      text: checkText
     }
+  }
 
-    return checkRunPayload;
+  return checkRunPayload;
+}
+
+export function getConfigForTable(widths: number[]): any {
+  let config = {
+    columns: {
+      0: {
+        width: widths[0],
+        wrapWord: true
+      },
+      1: {
+        width: widths[1],
+        wrapWord: true
+      },
+      2: {
+        width: widths[2],
+        wrapWord: true
+      },
+      3: {
+        width: widths[3],
+        wrapWord: true
+      }
+    }
+  };
+
+  return config;
+}
+
+export function extractErrorsFromLogs(outputPath: string) {
+  const out = fs.readFileSync(outputPath, 'utf8');
+  const lines = out.split('\n');
+  lines.forEach((line) => {
+    const errIndex = line.indexOf("FATAL");
+    if (errIndex >= 0) {
+      const err = line.substring(errIndex);
+      core.error(err);
+    }
+  });
+}
```

Comment:
Return errors array maybe? And add the logic of writing to `core.error` at the invoker.

Reply:
Done